pimd/pim6_cmd.c

```diff
@@ -300,6 +300,57 @@ DEFUN_HIDDEN (interface_no_ipv6_pim_ssm,
 			"frr-routing:ipv6");
 }
 
+DEFUN_HIDDEN (interface_ipv6_pim_sm,
```

Comment:
Hidden commands are not required. It is only for backward compatibility which we don't have currently.